Source: Upstream homepage, already applied.
Reason: Can't login after protocol change, bug #230387
--- src/icqpacket.cpp
+++ src/icqpacket.cpp
@@ -821,7 +821,7 @@
 
   // Static versioning
   buffer->PackUnsignedLongBE(0x00160002);
-  buffer->PackUnsignedShortBE(0x010A);
+  buffer->PackUnsignedShortBE(0x010B);
   // Client version major (4 == ICQ2000, 5 == ICQ2001)
   buffer->PackUnsignedLongBE(0x00170002);
   buffer->PackUnsignedShortBE(0x0014);
